lib/ansible/modules/packaging/language/pip.py

```diff
@@ -250,7 +265,7 @@ def _get_full_name(name, version=None):
     if version is None or version == "":
         resp = name
     else:
-        resp = name + '==' + version
+        resp = "\"%s %s\"" % (name, _version_number_to_formula(version))
```

Comment:
This syntax with parens isn't supported in outdated pip

Reply:
Why change the old format at all? That works on all relevant pip versions and also allows multiple versions joined by comma.

Reply:
@robinro Sorry I'm not sure which old format you mentioned. 
If you mean this one `name + '==' + version`, I think it doesn't support the case `version` is a formula like `>=1,11,1,<=1.12.1`. Here the new format make package and version formula become like `"django ==1.11.1"` or `"django >=1.11.1,<=1.12.1,!=1.11.3"` which can be parsed by pip.

Reply:
It's probably because of how distutils' predicate parser work, that's why we need to have a good distribution requirement representation.